### ISSUE TITLE
ST fills preseeded device with tmp files, probably when copy_file_range is set.

### ISSUE THREAD

**Issue by JNKL** [syncthing/syncthing#9856]
### What happened?

I have a preseeded device (pretty much everything is already there, around 3 TiB).
`<copyRangeMethod>copy_file_range</copyRangeMethod>` is set on all folders.
When I start syncing the device, nothing is downloaded, as it should, because the files are already there but ST writes a tmp file for every "transfer"/sync, which fills the volume until ST stops syncing because more than 99% space is used.
`btrfs fi du FILE` says the tmp files don't share any space. So the volume gets indeed filled.

```
# find -type f -name *syncthing*tmp|wc -l
723055
```

```
# btrfs fi df ./
Data, single: total=3.57TiB, used=3.57TiB
System, DUP: total=8.00MiB, used=416.00KiB
Metadata, DUP: total=15.00GiB, used=7.27GiB
GlobalReserve, single: total=512.00MiB, used=48.00KiB
```

```
# find -type f -name *syncthing*tmp -print0| du -hc --files0-from=-
[..]
520G    insgesamt
```

Until I run `find -type f -name *syncthing*tmp -delete` ST stops at "Preparing to Sync".

### Syncthing version

v1.28.1

### Platform & operating system

Linux

**Comment by JNKL** [comment]
`copy_file_range` is not the problem. Same behaviour with `standard`.

**Comment by JNKL** [comment]
Calculated checksums with deephash. Both drives have exactly the same file content. I think I will give up.

**Comment by calmh** [comment]
Is the other side a very old setup?

**Comment by JNKL** [comment]
No. Both were new. I copied the data (preseeded the drive) with `rsync`. Took the drive to another site and installed it in a server. Set up the server. Set up the folder at the server, waited initial scan to finish, shared it with home and then ST scanned the drive at home, started "syncing" and filled the drive with tmp files. "Sync" never finished because ST filled up the drive until it was full and stopped sync therefor.
